### PR TITLE
create Cucumber scenario

### PR DESCRIPTION
**Pull request by gokhakan**
### What changed
We have added a module called : integration-tests
We have created a basic setup to test session endpoint

### Why did it change

This test will act as a QA gateway.

### Issue tracking
 
https://govukverify.atlassian.net/browse/OJ-994

### To run the test

`gradle cucumber`

### CODE REVIEW

**Review by techaman-m — DISMISSED** [review]
LGTM

File: integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/SessionEndpointSteps.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class SessionEndpointSteps {
+
+    private String userIdentityJson;
+    private String ADDRESS_CRI_DEV = "address-cri-dev";
```
Comment (techaman-m) [inline integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/SessionEndpointSteps.java:26]: Note: We still need a `cri` for core stub

File: integration-tests/build.gradle [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	jacksonDatabind = '2.9.5'
+	apacheHttpClient = '4.5.13'
+	apacheCommonLang = '3.12.0'
+}
+
+dependencies {
+	testImplementation "io.cucumber:cucumber-java:$cucumberVersion"
+	testImplementation "io.cucumber:cucumber-junit:$cucumberVersion"
+	testImplementation "org.apache.httpcomponents:httpclient:$apacheHttpClient"
+	testImplementation "org.apache.commons:commons-lang3:$apacheCommonLang"
```
Comment (techaman-m) [inline integration-tests/build.gradle:27]: Do we need the library `apache.commons`

Comment (techaman-m) [inline integration-tests/build.gradle:27]: I dont think we need the library `apache.commons-lang3` ?

**Comment by jkunle** [comment]
I would have stopped, at having just the basic setup on this PR without Session feature / Session test or they they should have been dummy files. to be implemented in a later PR

**Review by techaman-m — APPROVED** [review]

**Pull request merged**